R/utilities-results.R

```diff
@@ -0,0 +1,57 @@
+#' Saves the simulation results to csv file
+#'
+#' @param results Results to export (typically calculated using \code{runSimulation} or imported from file)
+#' @param simulation Instance of a simulation used to calculate the results
+#' @param fileName Full path where the results will be saved.
+#'
+#' @examples
+#' simPath <- system.file("extdata", "simple.pkml", package = "ospsuite")
+#'
+#' # Load the simulation
+#' sim <- loadSimulation(simPath)
+#'
+#' # Add some outputs to the simulation
+#' addOutputs("Organism|**|*", sim)
+#'
+#' # Run the simulation
+#' results <- runSimulation(sim)
+#'
+#' # Export the results to csv file
+#' exportResultsToCSV(results, sim, tempfile())
+#' @export
+exportResultsToCSV <- function(results, simulation, fileName) {

```

Comment:
Same here - rename `fileName` to `outputCSVname` or even `outputCSVpath`?

Reply:
I don't like having acrnonyn like CSV in the name of the parameters. What about `outputFilePath` ? That way we can be consistent and use this allover

Reply:
or we specify what the file is about (not the extension)
resultsFilePath, simulationFilePath, pkanalysesFilePath etc...

Reply:
or we follown ggplot. and call it filePath (does nto matther if it;s import export etc..)
the context is clear and there is doc
I'd vote for `filePath` to be hontest

Reply:
Yeah OK, I can live with `filePath` :)